docs/release-notes/release-notes-0.20.0.md

```diff
@@ -25,6 +25,10 @@
 ## Functional Enhancements
 
 ## RPC Additions
+* When querying [`ForwardingEvents`](https://github.com/lightningnetwork/lnd/pull/9813)
+logs, the response now include the incoming and outgoing htlc indices of the payment 
+circuit. The indices are only available for forwarding events saved after v0.20.
```

Comment:
I think we might move 9813 to 20 ?

Reply:
No, the other way around. We moved it from 0.20 to 0.19.2, since it doesn't include a migration.